Maquetado/views/bookList.html

```diff
@@ -9,9 +9,9 @@
   <link rel="stylesheet" type="text/css" href="../application.css">
 </head>
 
-<body class='bookListBody'>
+<body>
   <header class='header'>
-      <img src='../resources/img/LogoWolox.png' />
+      <img src='../resources/img/LogoWolox.png' class='woloxLogoImage'/>
```

Comment:
`alt` attribute is missing.
the alt attribute is very important for the accessibility.